oaknational/Oak-Web-Application
# PUPIL-678 feat(lesson-listing): style lesson listing page

## Description

Music year: [2007](https://www.youtube.com/watch?v=PsO6ZnUZI0g)
Music year: [1979](https://www.youtube.com/watch?v=fiD39jo5Yo4)


Add style to Lesson lIsting pages

- creating a view file to create the UI
- use oak-components to build UI

### Acceptance criteria

- [x]  page matches designs
- [x]  passing tests


## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2412--oak-web-application.netlify.thenational.academy/pupils/beta/programmes/maths-secondary-year-11-higher-l/units/further-algebraic-fractions-2758/lessons
2. Click on lesson or back button
3. You should see all the available legacy lessons; note that the lesson listing is not yet navigable from the actual lesson pages

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
